itest/litd_custom_channels_test.go

```diff
@@ -4717,6 +4752,8 @@ func testCustomChannelsSelfPayment(ctx context.Context, net *NetworkHarness,
 		fmt.Sprintf(node.ListenerFormat, alicePort),
 	))
 
+	litdArgs = append(litdArgs, litdArgNoopHTLCs...)
```

Comment:
This is executed in a backward compatibility test as well, so we have to do this conditionally. Or perhaps it's easier to just add it conditionally in `HarnessNode.overrideFlagsAndBinary()`, then the diff of this commit also becomes smaller and we don't have to think to add it to new tests either.